src/ts/uiconfig.ts

```diff
@@ -90,4 +90,10 @@ export interface UIConfig {
    * Default: true
    */
   enableSeekPreview?: boolean;
+  /**
+   * Specifies if the player should be set to fullscreen by clicking on the `PlaybackToggleButton` or
+   * `HugePlaybackToggleButton` when attempting the initial playback start.
```

Comment:
Pls also mention the overlay

Reply:
[nit] maybe `Specifies if the player should enter fullscreen by...`

Reply:
Updated in 94bbd241eb4118f9432bda2c2320fa4999116aa8